.github/workflows/build-sakura.yml

```
@@ -75,8 +75,44 @@ jobs:
       shell: cmd
 
     ## see https://github.com/actions/upload-artifact
-    - name: Upload
+    - name: Upload Installer
       uses: actions/upload-artifact@v2
       with:
-        name: exe ${{ matrix.platform }} ${{ matrix.config }}
-        path: '*.zip'
+        name: Installer ${{ matrix.platform }} ${{ matrix.config }}
+        path: 'sakura-*-Installer.zip'
+
+    - name: Upload Installer MD5
+      uses: actions/upload-artifact@v2
+      with:
+        name: Installer MD5 ${{ matrix.platform }} ${{ matrix.config }}
+        path: 'sakura-*-Installer.zip.md5'
+
+    - name: Upload Exe
+      uses: actions/upload-artifact@v2
+      with:
+        name: Exe ${{ matrix.platform }} ${{ matrix.config }}
+        path: 'sakura-*-Exe.zip'
+
+    - name: Upload Exe md5
```

```suggestion
    - name: Upload Exe MD5
```
インストーラーと表記をあわせた方がよさそう。

---

対応しました。